contracts/src/v0.8/dev/KeeperRegistryCheckUpkeepGasUsageWrapper.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+import "../KeeperRegistry.sol";
+import "../ConfirmedOwner.sol";
+
+/**
+ * @notice This contract serves as a wrapper around a keeper registry's checkUpkeep function.
+ */
+contract KeeperRegistryCheckUpkeepGasUsageWrapper is ConfirmedOwner {
+  KeeperRegistryExecutableInterface private immutable i_keeperRegistry;
+
+  /**
+   * @param keeperRegistry address of a keeper registry
+   */
+  constructor(KeeperRegistryExecutableInterface keeperRegistry) ConfirmedOwner(msg.sender) {
+    i_keeperRegistry = keeperRegistry;
+  }
+
+  /**
+   * @return the keeper registry
+   */
+  function getKeeperRegistry() external view returns (KeeperRegistryExecutableInterface) {
+    return i_keeperRegistry;
+  }
+
+  /**
+   * @notice This function is called by monitoring service to estimate how much gas checkUpkeep functions will consume.
+   * @param id identifier of the upkeep to check
+   * @param from the address to simulate performing the upkeep from
+   */
+  function checkUpkeep(uint256 id, address from)
+    external
+    returns (
+      bool checkUpkeepSuccess,
+      bytes memory performData,
+      uint256 gasUsed
```

Comment:
these variables look unused - can we remove them?

Reply:
so i discover that when these return values are named, i can use response.checkUpkeepSuccess or response.gasUsed in our tests. Without named return values, we can only treat the response in test as an array and response[0] is the boolean, response[1] is the performData etc.
in this case, i think it makes sense to optimize the contract at a small cost of test readibility.